app/utilities/im_processing.py

```diff
@@ -6,6 +6,7 @@
 from PIL import Image, ImageTk  # type: ignore[import]
 
 
+# Vs code is showing me this error: Union requires two or more type arguments.
```

Comment:
Good to know! I'm using a simpler IDE and it didn't say anything here.
Could you remove this comment and do the necessary changes? I found some information [here](https://stackoverflow.com/a/52471875)

Reply:
I ve changed it, lets see.